### PR TITLE
Vaishali/home

### PR DESCRIPTION
**Pull request by VaishaliGovindaraj**
Added code to Index file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,8 +69,96 @@ <h1 class="header__page-title">
             </div>
             <div class="main-content">
                 <!--Here we will put our content, erase the filler class and div-->
```
Comment (andrewkalumba) [inline index.html:71]: don't forget to remove this line

File: index.html [hunk 2]
```diff
@@ -68,8 +69,96 @@ <h1 class="header__page-title">
...
-                <div class="filler">
-                  
+                <div class="home">
+                    <section class="about-git">
+                        <h3>About GIT</h3>
```
Comment (TobiasKjernell) [inline index.html:74]: make a class for this h3 so it doesnt collide with other h3 in layout.

File: index.html (hunk 1 above)
Comment (VaishaliGovindaraj) [reply index.html:71]: Ok i will look into it

File: styles/index.css [hunk 3]
```diff
@@ -24,4 +22,96 @@ body {
...
-}
\ No newline at end of file
+}
+
+.home{
+    display:flex;
+    flex-direction:column;
+    min-height:100%;
+}
+
```
Comment (TobiasKjernell) [inline styles/index.css:32]: white spacing between classes

Comment (VaishaliGovindaraj) [reply styles/index.css:32]: Sure

File: styles/index.css [hunk 4]
```diff
@@ -24,4 +22,96 @@ body {
...
+.grid_box:hover{
+    transform: perspective(1000px) scaleZ(6) rotateZ(40deg); 
+    width: 75%;
+    font-size: 1.5rem;
+    background-color: beige;
+    z-index:100;
+}
+
```
Comment (andrewkalumba) [inline styles/index.css:103]: check on the paragraphs spaces

File: styles/index.css [hunk 5]
```diff
@@ -24,4 +22,96 @@ body {
...
+.home{
+    display:flex;
+    flex-direction:column;
+    min-height:100%;
+}
+
+
+h3{
```
Comment (TobiasKjernell) [inline styles/index.css:34]: this should be a class instead

File: index.html (hunk 1 above)
Comment (VaishaliGovindaraj) [reply index.html:71]: Removed the line

File: index.html (hunk 2 above)
Comment (VaishaliGovindaraj) [reply index.html:74]: class for h3 has been done

File: styles/index.css (hunk 3 above)
Comment (VaishaliGovindaraj) [reply styles/index.css:32]: White space removed

**Review by VaishaliGovindaraj — COMMENTED** [review]
Resolved the comments hightlighted

**Pull request merged**